app/scripts/modules/core/src/help/help.contents.ts

```diff
@@ -113,7 +113,7 @@ module(HELP_CONTENTS, [])
     'pipeline.config.docker.bake.targetImage': '<p>The name of the resulting docker image.</p>',
     'pipeline.config.docker.bake.targetImageTag': '<p>The tag of the resulting docker image, defaults to commit hash if available.</p>',
     'pipeline.config.docker.bake.organization': '<p>The name of the organization or repo to use for the resulting docker image.</p>',
-    'pipeline.config.bake.baseAmi': '<p>(Optional) ami-????????</p>',
+    'pipeline.config.bake.baseAmi': '<p>(Optional) If Base AMI is specified, this will be used instead of the Base OS provided',
```

Comment:
Should this say "Base image" instead of "Base OS"?

Reply:
Third field for 'Bake Configuration' form says 'Base OS'. So if we specify this field 'base AMI', bake step ignores what we have provided in 'Base OS field'. That has been my understanding till now and conveying this in the tooltip

Reply:
Ah, right, makes sense.